### PR TITLE
Complete todo list

### PR DESCRIPTION
**Pull request by binodbhusal**
- Added to-do List functionality and created the module.
- Added HTML CSS for the to-do list.
- Completed all required functionality of the to-do list.

### CODE REVIEW

File: modules/functions.js
Comment (yashodhicy) [inline modules/functions.js:1]: Excellent work on this code. You have used arrow functions which is a good practice in js. It is also better to use ES classes to make this code cleaner.

**Review by yashodhicy — CHANGES_REQUESTED** [review]
Hi @binodbhusal , You have done an excellent job so far. 


👍🏻 used HTML and CSS best practices
👍🏻 have used good variable names and appropriate arrow functions
👍🏻 have a clean code

but there are a few changes that need to be done to improve your beautiful code. look at the comment section for the requested change.

File: modules/functions.js [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+
+  const btnRemove = document.querySelectorAll('.btnremove');
+  if (btnRemove) {
+    btnRemove.forEach((btn, index) => {
+      btn.addEventListener('click', () => {
+        removeTask(index);
+      });
+    });
+  }
+  const editTask = (index) => {
```
Comment (yashodhicy) [inline modules/functions.js:38]: Excellent work on this code. You have used arrow functions which is a good practice in js. It is also better to use ES classes to make this code cleaner.`

**Review by yashodhicy — APPROVED** [review]
great job so far.You are ready to go for next step. Congratulations
congrats

File: modules/functions.js (hunk 1 above)
Comment (binodbhusal) [reply modules/functions.js:38]: @yashodhicy Thank you so much!!

**Pull request merged**